riscv-rt/src/lib.rs

```diff
@@ -1,126 +1,44 @@
-//! Minimal startup / runtime for RISC-V CPU's
+//! Startup code and minimal runtime for RISC-V microcontrollers
```

Comment:
Do these changes imply incompatibility with application SoCs with multiple HARTs?

Why the change from `CPU` to `microcontroller`?

Reply:
I just copy/pasted the docs in `cortex-m-rt`, the liker should present the exact same configuration as before